.github/workflows/ci-windows-shared-local.yml

```diff
@@ -0,0 +1,59 @@
+# Do a shared library build with local dependencies.
+
+name: CI Windows Shared Local
+on: [push, pull_request]
```

Comment:
Is it intentional to run it everytime or should it be reserved to run on merge only?  
The intent is to avoid a long list of presubmit in the PR GitHub UI and to save resources.

Reply:
I didn't consider this issue. I copied this line from some other file (most likely ci-windows-installed.yml).

Please advise what we should use here. Thanks!

Reply:
For example:

https://github.com/AOMediaCodec/libavif/blob/676a6cb82c0261cf5474c086f8e94f0f65ad4645/.github/workflows/ci-unix-static.yml#L3-L9

Reply:
Done. Thanks!